src/docs/community/rebrand.mdx

```diff
@@ -0,0 +1,27 @@
+import { Divider, FAQExpander } from '../../app/components';
+
+<!-- KEYWORDS: faq question answers -->
+
+# PX Blue is Rebranding
+
+In Q4 of 2021, we will be rebranding our design system from PX Blue to Brightlayer UI. This rebranding effort will impact all resources that PX Blue currently provides, including:
```

Comment:
From Power Xpert Blue ("PX Blue") to Brightlayer User Interface ("Brightlayer UI" or "BLUI").

Reply:
Nice, added.